database/postgres/pg-auth.js

```diff
@@ -0,0 +1,23 @@
+const dev = {
```

Comment:
you should have this inside of your git ignore and create an example.pg-auth.js file to upload instead for best practices

Reply:
good idea, thanks!